src/PluginInterface.cpp

```diff
@@ -219,8 +219,8 @@ void PluginAtom::retrieveCached(const Query& query, Answer& answer)
   //
   // The most efficient way would be:
   // * use cache for same inputSet + same *inputi + more specific pattern
-  // * store new cache for new inputSet/*inputi combination or unrelated (does not unify) pattern
-  // * replace cache for existing inputSet/*inputi combination and less specific (unifies in one direction) pattern
+  // * store new cache for new inputSet inputi combination or unrelated (does not unify) pattern
```

Comment:
Please write `inputSet / *inputi` to solve this issue (also below). The `*` is important.